lib/nrf_modem_lib/Kconfig.modemlib

```diff
@@ -32,11 +32,24 @@ config NRF_MODEM_LIB_TRACE_ENABLED
 
 if NRF_MODEM_LIB_TRACE_ENABLED
 
+config NRF_MODEM_LIB_TRACE_THREAD_PROCESSING
+	bool "Process modem traces in a thread [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  Use a normal priority thread to forward modem traces to the trace medium instead of forwarding
+	  in interrupt context. This allows for asynchronous operations, like async UART. A small amount
```

Comment:
```suggestion
          Use a thread to forward modem traces to the trace medium instead of doing so
	  in interrupt context. This allows for asynchronous operations, like async UART. A small amount
```